### PR TITLE
Update composer.json and README.md

### PR DESCRIPTION
**Pull request by rvanlaak**
Update composer for Symfony3. Add quotes to config path, also fix line length.

Can you also release new version?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,15 +22,14 @@
...
-        "symfony/console": "2.*",
+        "symfony/framework-bundle": "^2.7|^3.0",
+        "symfony/console": "^2.7|^3.0",
         "maxmind-db/reader": "0.3.*"
     },
     "require-dev": {
         "phpunit/phpunit": "*",
         "composer/composer": "*@alpha"
     },
-    "minimum-stability": "stable",
```
Comment (mente) [inline composer.json:33]: why did you remove stability?

File: README.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ Insomnia MaxGeoIp Bundle
 Installation
 ====
 
-This bundle helps you deliver MaxMind GeoIp functionality inside your symfony application. Based on [MaxMind library](https://github.com/maxmind/MaxMind-DB-Reader-php).
+This bundle helps you deliver MaxMind GeoIp functionality inside your symfony application. Based 
+on [MaxMind library](https://github.com/maxmind/MaxMind-DB-Reader-php).
```
Comment (mente) [inline README.md:10]: why did you put it on other line?

File: composer.json (hunk 1 above)
Comment (rvanlaak) [inline composer.json:33]: Basically because that's not relevant for bundles, and should only be included in your own project. It is a root-only field, as you can see here: https://getcomposer.org/doc/04-schema.md#minimum-stability

File: README.md [hunk 3]
```diff
@@ -64,11 +67,12 @@ Add following to yours `composer.json`:
 }
 ```
 
-Extra part `maxmind-db-path` is required and should be same as `insomnia_max_mind_geo_ip.path` in symfony configuration. That's it! You're ready to start
-
+Extra part `maxmind-db-path` is required and should be same as `insomnia_max_mind_geo_ip.path` in symfony 
+configuration. That's it! You're ready to start.
```
Comment (mente) [inline README.md:71]: why another line?

File: README.md [hunk 4]
```diff
@@ -30,25 +32,26 @@ Configuration is pretty straight forward:
...
 ```
 
-Command will save and unpack database to destination specified in configuration. So you don't need to worry about correct placement.
+Command will save and unpack database to destination specified in configuration. So you don't need to worry 
+about correct placement.
 
-**Hint**: if you want to automate database update, you can integrate it in your composer install/update phase using provided script handler.
-Add following to yours `composer.json`:
+**Hint**: if you want to automate database update, you can integrate it in your composer install/update phase 
+using provided script handler. Add following to yours `composer.json`:
```
Comment (mente) [inline README.md:54]: again new line

File: README.md (hunk 2 above)
Comment (rvanlaak) [inline README.md:10]: Because it exceeds soft line limit of 110 chars everywhere :)

File: composer.json [hunk 5]
```diff
@@ -22,15 +22,14 @@
         ]
     },
     "require": {
-        "symfony/framework-bundle": "2.*",
-        "symfony/console": "2.*",
+        "symfony/framework-bundle": "^2.7|^3.0",
```
Comment (mente) [inline composer.json:25]: why 2.7?

Comment (rvanlaak) [inline composer.json:25]: Agree, should be `2.8` because that's the LTS version. Thought it was `2.7` up till now, see: http://symfony.com/download

Comment (mente) [inline composer.json:25]: 2.\* might be 2.1 as well. Guess bumping up requirements just because it's new LTS doesn't make sense. I'm not sure but guess this bundle still can work with symfony 2.3. Needs checking

Comment (rvanlaak) [inline composer.json:25]: You are correct, but composer will just install the old version of this bundle which still is compatible.

Comment (mente) [inline composer.json:25]: i guess everyone wants to install newest version, not old one ;)

**Comment by mente** [comment]
Thanks for the contribution but I have to close it. I've incorporated most important changes like adding symfony 3.0 support. Also included Travis builds for all versions.

**Comment by rvanlaak** [comment]
Perfect, yes I see https://github.com/mente/MaxMindGeoIpBundle/commit/1eaf979ae202a4f53a2d6d93eb0139444af9be92 and https://github.com/mente/MaxMindGeoIpBundle/commit/98301b1d291d750e75c72b9ad4c3b5f4cfda843c fix it! :+1:
